docs/design/3_2_multirotor.tex

```diff
@@ -262,8 +245,7 @@ \subsubsection{Propulsion System}
 
 \paragraph{Speed Controllers}
 
-The electronic speed controller (ESC) controls the voltage and current supplied to a BLDC motor. All ESCs 
-function identically and the design parameters are size/weight, rating, efficiency, and cost.
+The electronic speed controller (ESC) controls the voltage, current, and phase supplied to a BLDC motor. Since BLDC motors function by rotating magnetic field using electric commutation, the ESCs need to precisely control the timing and phase of the input to the three phase lines. All ESCs function similarly and the design parameters are size/weight, power rating, power efficiency, and cost.
```

Comment:
```suggestion
The electronic speed controller (ESC) controls the voltage, current, and phase supplied to a BLDC motor. Since BLDC motors function by rotating a magnetic field through electric commutation, the ESCs need to precisely control the timing and phase of the input to the three phase lines. All ESCs function similarly, with the pertinent design parameters being size/weight, power rating, power efficiency, and cost.
```
"Phase lines"?